src/style.css

```
@@ -1 +1,103 @@
 /* Styles go here */
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+  font-family: "Roboto", sans-serif;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+}
+
+.header {
+  width: 1200px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
```

Create a variable for the colors
```suggestion
  background-color: var(--color-white);
```